src/shared/ui-kit/components/Stepper/Stepper.tsx

```diff
@@ -22,42 +20,47 @@ const GetStatusIcon = (iconType: string) => {
 
 interface StepperUIProps {
   states: ProposalState[];
-  orientation?: Orientation;
 }
 
-function StepperUI(props: StepperUIProps) {
-  const { orientation = "vertical", states } = props;
+export function Stepper(props: StepperUIProps) {
+  const { states } = props;
+  const isLastStep = states.length - 1;
   return (
-    <Box sx={StepperStyles} marginLeft="-10px">
-      <Stepper orientation={orientation}>
-        {states.length !== 0 &&
-          states.map((state) => {
-            return (
-              <Step key={state.status}>
-                <Flex flexDirection="row" gap="13.5px">
-                  {GetStatusIcon(state.iconType)}
-                  <StepLabel>
-                    <Flex flexDirection="row" gap="4px">
-                      <Text
-                        textStyle="b4"
-                        color={state.iconType === ProposalStateIcon.Disabled ? Color.Grey_01 : Color.Text_Primary}
-                      >
-                        {state.status}
-                      </Text>
-                      {state.timeRemaining && (
-                        <Text textStyle="b2" color={Color.Grey_02}>
-                          {` - ${state.timeRemaining}`}
-                        </Text>
-                      )}
-                    </Flex>
-                  </StepLabel>
-                </Flex>
-              </Step>
-            );
-          })}
-      </Stepper>
-    </Box>
+    <ChakraStepper index={0} orientation="vertical" gap="0" padding="1.5rem 1rem">
+      {states.map((step, index) => (
+        <Step key={step.status}>
+          <Flex direction="column" gap="1">
+            <Flex flexDirection="row" gap="3">
+              {GetStatusIcon(step.iconType)}
+              <Flex flexDirection="row" gap="0.3rem" alignItems="center">
+                <Text
+                  textStyle="b4"
```

Comment:
Please use updated `textStyles` and `colors` for the Stepper.

Reply:
done